src/main/java/cz/muni/fi/pa165/bluebat/entity/Excursion.java

```diff
@@ -56,7 +59,7 @@
     @OneToMany
     @OrderBy("validFrom DESC")
     @JoinTable
-    private List<Price> prices = new ArrayList<>();
+    private final List<Price> prices = new ArrayList<>();
```

Comment:
Proč jsi dal prices jako final? 🤔

Reply:
Tak počítám, že přepisovat referenci by asi nikde moc nemělo smysl, takže mi to takhle přijde takové hezčí. A taky mě štvalo, že na to IDE hází upozornění :D